### DISCUSSION TITLE
Code Block is really Ugly.

### DISCUSSION THREAD

**Discussion by blauer4** [Zettlr/Zettlr#4199] [General]
Hi,
I found this really cool md editor, and I like it quite a lot. But there are some things that I don't like, like the code block:
![image](https://user-images.githubusercontent.com/56732985/225035193-7bc1e987-9bd6-4108-bec1-9b9bad98a78e.png)

As you can see it has this weird behaviour that it cuts part of the initial block. I really don't understand why and it doesn't seem to have a way to fix it. I'm using the 3.0.0beta2 version. 
I'm coming from obsidian, but there were really a lot of things that bothered me from that editor. The fact is that I like it more when it comes to cleanliness of the WYSIWYG editor. 
In this regard, I prefer the block code in obsidian that is like this: 
![image](https://user-images.githubusercontent.com/56732985/225035363-d488ec73-fc65-4d4e-8f4e-db426adbfec8.png)

And this is what md is like:
![image](https://user-images.githubusercontent.com/56732985/225035477-7888d247-229e-4300-8e23-61aa37fcbdb5.png)

So I would ask if there is a plan to improve the code block implementation, and in general the rendering of the WYSIWYG part.

Thanks a lot!

**Comment by nathanlesage** [comment]
I think I've fixed this recently, so it could be that it didn't yet make it into any beta. You can check out the nightly, if you want to, and if that doesn't fix it, this could well be a bug.

**Reply by blauer4** [reply]
Oh, this is perfect. I Didn't see the nightly versions because I took the beta from GitHub. Do you have a timeline for when you will release a new stable version?

**Reply by nathanlesage** [reply]
Not really … I want to make the next stable version really good, and I'd rather take longer than rush it. But, if you'd like to you can definitely stick to the nightlies, I do not plan on introducing any unstable things anymore, so it should only get more stable. And then once the 3.0 is out, you can switch to the stables once again :)

**Reply by 87113** [reply]
Hi, just to say I saw this buggy rendering in the beta3. 

So thank you that you already work on it! :)

**Comment by blauer4** [comment]
Mhhh I checked out the nightly and it seems to work. One thing is that old files that had that problem still have that rendering issue. I don't Know why. 
Here is an example: 
![image](https://user-images.githubusercontent.com/56732985/225583805-36a6e296-d6c2-4ada-a323-e1d4d46e15b5.png)
The first block code has a space before the ending code block tag. The second is just the plain code block autocompletion. Any Idea why?

**Reply by nathanlesage** [reply]
This sounds definitely like a bug! Let me quickly transfer this to an issue …
